packages/hierarchies-react/src/presentation-hierarchies-react/UseTree.ts

```diff
@@ -117,10 +117,10 @@ interface ReloadTreeOptions {
 
 /** @public */
 export interface UseTreeResult {
-  /**
-   * Array containing root tree nodes. It is `undefined` on initial render until any nodes are loaded.
-   */
-  rootNodes: PresentationTreeNode[] | undefined;
+  /*   * Array containing root tree nodes. It is `undefined` on initial render until any nodes are loaded. */
+  rootNodes: PresentationHierarchyNode[] | undefined;
+  /** An error object which is defined when root nodes fail to load. Defined as undefined when rootNodes where sucessfully loaded or are in loading process */
+  rootError: PresentationError | undefined;
```

Comment:
It sounds a bit strange: `Defined as undefined`, don't you think?
```suggestion
  /*   * Array containing root tree nodes. It is `undefined` on initial render until any nodes are loaded. */
  rootNodes: PresentationHierarchyNode[] | undefined;
  /** An error object which is defined when root nodes fail to load. It is `undefined` when rootNodes where sucessfully loaded or are in loading process */
  rootError: PresentationError | undefined;
```

Reply:
Changed